resources/guis.yml

```diff
@@ -164,7 +164,7 @@ guis:
   - Linux
   price: Free
   license: MIT
```

Comment:
funny thing, i think this file has become a bit messy and some numbers are missing 😬  a problem to solve later

Reply:
It is quite difficult to maintain this list currently, a script that scrapes google trends and sorts automatically would be much better.
I had a brief look into this but all existing packages I found for this no longer work, so would require more time to do than I have right now.